src/main/java/com/redhat/lightblue/camel/LightblueProducer.java

```diff
@@ -36,4 +40,25 @@ public void process(Exchange exchange) throws Exception {
         }
     }
 
+    /*
+     * TODO I would prefer this method return a LightblueResponse instead of Object,
```

Comment:
Did you create an issue for both AbstractLightblueDataRequest and AbstractDataBulkRequest to implement the same base or interface?